test/suites/rpm/install-and-upgrade-successful.robot

```diff
@@ -68,6 +68,31 @@ Upgrade From Previous Version
     Wait Until Greenboot Health Check Exited
     [Teardown]    Clean Up Test
 
+Verify Service Restart On Upgrade
```

Comment:
This test takes almost 7 minutes and it's very similar to the test above.
What do you think about merging them?

Reply:
Good idea!